src/Main.js

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import "./Main.css"
+import "../node_modules/bootstrap/dist/css/bootstrap.min.css"
+import "../node_modules/bootstrap/dist/js/bootstrap.min.js"
+import "../node_modules/jquery/src/jquery.js"
+import "../node_modules/popper.js/dist/popper.min.js"
+
+
+class Navbar extends React.Component {
+
+  render() {
+    return (
+   <nav className="navbar navbar-expand-sm bg-dark navbar-dark">
+    <ul className="navbar-nav">
+      <li className="nav-item active">
+        <a className="nav-link" href="#">CollegePrep.org</a>
+      </li>
+      <li className="nav-item">
+        <a className="nav-link" href="#">Link</a>
+      </li>
+      <li className="nav-item">
+        <a className="nav-link" href="#">Link</a>
+      </li>
+      <li className="nav-item">
+        <a className="nav-link" href="#">Link</a>
+      </li>
+    </ul>
+</nav>
+    );
+  }
+}
+
+// separation of selection box --> avatar, person's name, testimony
+
+/* Avatar */
+class Avatar extends React.Component {
+  render () {
+    return (
+      <img className="circle avatar" src={this.props.img} />
+      )
+  }
+}
+
+/* Testimony */
+class Testimony extends React.Component {
+  render () {
+    return (
+      <div className="d-flex p-3 align-items-center">
+        <div className="p-2">
+          <Avatar img={this.props.img} />
+          <p align="center"> {this.props.name}, {this.props.grade} </p>
+        </div>
+        <div className="p-2">
+          <p> {this.props.content} </p>
+        </div>
+      </div>
+      
+      )
+  } 
+}
+
+
+/* Dropdown options */
+class DropdownPrev extends React.Component {
+  render () {
+    return (
+  <div className="dropdown">
+  <button type="button" className="btn btn-primary dropdown-toggle" data-toggle="dropdown">
+    Subject
+  </button>
+  <div className="dropdown-menu">
+    <a className="dropdown-item" href="#">ACT</a>
+    <a className="dropdown-item" href="#">SAT</a>
+  </div>
+</div> )
+  }
+}
+
+// TODO: continue exploring how to loop over all the options posted 
+// so that we can use Dropdown generically
+
+// class DropdownOption extends React.Component {
+//   render() {
+//     return <a href={this.props.link}>{this.props.text}</a>;
+//   }
+// }
+
+/* More generalized dropdown option */
+// class Dropdown extends React.Component {
+//   renderDropdownOption(option, link) {
+//     return <NavbarOption text={option} link={link} />;
+//   }
+//   renderDropdownTitle() {
+//        <button type="button" className="btn btn-primary dropdown-toggle" data-toggle="dropdown">
+//         Subject
+//       </button>
+//   }
+//   render () {
+//     return (
+//       <div>
+//       this.renderDropdownTitle();
+//       var options = {this.props.options};
+//       for (var i = 0; i < options.length; i++) {
```

Comment:
Instead use the .map function - see the link above. We're not using react-bootstrap per se, but this is a good example of how to use the map function.